lib/stack_master/parameter_resolvers/stack_output.rb

```diff
@@ -34,21 +35,41 @@ def cf
         @cf ||= StackMaster.cloud_formation_driver
       end
 
-      def validate_value!(value)
-        if !value.is_a?(String) || !value.include?('/')
-          raise ArgumentError, 'Stack output values must be in the form of stack-name/output-name'
+      def parse!(value)
+        if !value.is_a?(String) || !(match = @output_regex.match(value))
+          raise ArgumentError, 'Stack output values must be in the form of [region:]stack-name/output_name'
         end
+
+        [
+          match[:region] || cf.region,
+          match[:stack_name],
+          match[:output_name]
+        ]
       end
 
-      def find_stack(stack_name)
-        @stacks.fetch(stack_name) do
-          cf_stack = cf.describe_stacks(stack_name: stack_name).stacks.first
-          @stacks[stack_name] = cf_stack
+      def find_stack(stack_name, region)
+        unaliased_region = @config.unalias_region(region)
+        stack_key = stack_key(stack_name, unaliased_region)
+
+        @stacks.fetch(stack_key) do
+          regional_cf = cf_for_region(region)
```

Comment:
🤦‍♂️ You're right, it should have been `unaliased_region`. Fixed in a new commit.